components/dialog/RollbackWorkloadDialog.vue

```diff
@@ -165,6 +169,9 @@ export default {
       if (dialogs.length === 1) {
         dialogs[0].style.setProperty('--prompt-modal-width', width);
       }
+    },
+    sanitizeYaml(revision) {
```

Comment:
Small nit: this function is doing more than just generically sanitizing the yaml and could probably be renamed to something like "pickRevisionSpec" or even just use it directly in sanitizedSelectedRevision without making it it's own method.